src/main/scala/carldata/series/TimeSeries.scala

```diff
@@ -132,6 +133,18 @@ object TimeSeries {
     Iterator.iterate(start)(_.plusNanos(delta.toNanos)).takeWhile(_.isBefore(end)).toVector
   }
 
+  /** Find most frequent duration in time series */
+  def resolution[V: Numeric](ts: TimeSeries[V]): Duration = {
```

Comment:
This function doesn't require type V to be Numeric.

If we remove it we can also move it to TimeSeries class